sw/device/tests/BUILD

```diff
@@ -1760,6 +1760,28 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
+    name = "i2c_host_accelerometer_test",
+    srcs = ["i2c_host_accelerometer_test.c"],
+    cw310 = cw310_params(
+        tags = ["skip_in_ci"],  # Requires the Bob in CI.
```

Comment:
The "skip_in_ci" tag is a bit of a hack used to not run all ROM E2E tests that had many different permutations since they used different OTP images with different LC states. However they do get run in a nightly round.

If you want this to be run only with special HW hooked up, then maybe you should just tag this test with "manual" instead. Unless there is a specific CI machine in CI (i.e. Bob?) that can run this?

Reply:
@engdoreis @johngt do we want to run these in CI? Every board has a BoB connected.

Reply:
The intention is to run these tests in CI. All boards in CI, current and planned have a BoB either attached or ready to be attached to them.

Reply:
We will have the bitstream that enables the BoB in CI very soon, so I'll merge this PR and after the bitstream PR gets merged I'll file a PR removing this tag.